test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -2858,3 +2858,57 @@ def test_lowering_to_x86(self):
                 node_list,
                 lower=True,
             )
+
+    @skipIfNoX86
+    def test_annotate_mul_tensor(self):
+        class M1(torch.nn.Module):
+            def __init__(self, type):
+                super().__init__()
+                self.type = type
+
+            def forward(self, x, y):
+                if self.type == 0:
+                    return x * y
+                elif self.type == 1:
+                    return x * y.sum(-1)
+                elif self.type == 2:
+                    return x * y.sum()
+                else:
+                    return x * y.sum().item()
+
+        for type in [0, 1, 2, 3]:
```

Comment:
can we give a meaningful name with enum?

Reply:
Thanks for the comment. Enum may look better. But they are only used in this test case and their meanings can be inferred from the code, which is quite simple. Besides, it's difficult to give them concise names. So, maybe using numbers here is better. How does that sound to you?

Reply:
Sure, I'm fine with the current implementation, given that it's only used in the unit test.